appengine-standard-java8/kotlin-springboot-appengine-standard/pom.xml

```diff
@@ -68,7 +68,12 @@ limitations under the License.
     </dependency>
     <dependency>
       <groupId>org.jetbrains.kotlin</groupId>
-      <artifactId>kotlin-stdlib-jre8</artifactId>
+      <artifactId>kotlin-stdlib-jdk8</artifactId>
+      <version>${kotlin.version}</version>
```

Comment:
You might want to avoid using properties - DPE bot doesn't perform upgrades on them if you do.